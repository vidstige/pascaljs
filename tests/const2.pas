program Const2;
type
  TRgb = record
    r, g, b: Byte;
  end;
const
  I: Integer = 123;
  S: String = 'hej';
  A: array[0..3] of Word = (1, 16, 256, 4096);
  HEX10 = $10;
  HEX3f = $3f;
  HEX3F = $3F;
  BLACK: TRgb = (r:$00; g:$00; b:$00);
begin
  WriteLn('I = ', I);
  WriteLn('S = ', S);
  WriteLn('A[1] = ', A[1]);
  WriteLn('HEX10 = ', HEX10);
  WriteLn('HEX3f = ', HEX3f);
  WriteLn('HEX3F = ', HEX3F);
end.