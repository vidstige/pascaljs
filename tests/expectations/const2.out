I = 123
S = hej
A[1] = 16
HEX10 = 16
HEX3f = 63
HEX3F = 63
